src/use-async-task-fetch.js

```diff
@@ -6,9 +6,10 @@ import { useAsyncRun } from './use-async-run';
 const defaultInit = {};
 const defaultReadBody = (body) => body.json();
 
-const createFetchError = (message, responseBody) => {
-  const err = new Error(message);
+const createFetchError = (response, responseBody) => {
+  const err = new Error(`${response.statusCode} ${response.statusText}`);
```

Comment:
status instead statusCode

Reply:
https://developer.mozilla.org/en-US/docs/Web/API/Response/status

Oops! Please note, my recommendation is to use `err.response.status` instead of `err.message`, which is rather for logging.